async.md

```diff
@@ -220,4 +221,315 @@ Example
 		})
 ~~~
 
+# FileSystem 
 
+Class: ClearBladeAsync.FS()
+
+This class adds support for interacting with the sync’d filesystem. All file paths are relative to bucket root, except on an edge where you can use full paths.
+
+To instantiate the async filesystem class just call:
+
+~~~javascript
+	var FS = ClearBladeAsync.FS()
+~~~
+
+## ClearBladeAsync.FS(deployment_name)
+
+Represents a sync'd filesystem.
+
+* @param {string} deployment_name
+* @returns {FS}
+Example
+
+~~~javascript
+	var FS = ClearBladeAsync.FS()
+	
+~~~
+
+## ClearBladeAsync.FS.readDir(path)
+
+ This function reads the entire contents of a file.
+ If encoding is not specified, contents will be a `UInt8Array`. If encoding is specified, contents will be a string.
+
+ * @param {string} path
+ * @param {string} encoding
+ * @returns {Promise} -  file contents
+
+Example
+
+~~~javascript
+	var FS = ClearBladeAsync.FS()
+	FS.readDir(path)
+		.then(resp.success)
+		.catch(function(reason){
+			resp.error("caught: "+reason.message);
+		})
+~~~
+
+## ClearBladeAsync.FS.readFile(path,[encoding])
+
+ This function reads the entire contents of a file.
+ If encoding is not specified, contents will be a `UInt8Array`. If encoding is specified, contents will be a string.
+
+ * @param {string} path
+ * @param {string} encoding
+ * @returns {Promise} -  file contents
+
+Example
+
+~~~javascript
+	var FS = ClearBladeAsync.FS()
+	FS.readFile(path,[utf8]
+		.then(resp.success)
+		.catch(function(reason){
+			resp.error("caught: "+reason.message);
+		})
+~~~
+
+## ClearBladeAsync.FS.writeFile(path,data)
+
+This function writes the given data to a file, replacing the file if it already exists.
+
+ * @param {string} path
+ * @param {string|Uint8Array} data
+ * @returns {Promise}
+
+Example
+
+~~~javascript
+	var FS = ClearBladeAsync.FS()
+	FS.readFile(path,[utf8]
```

Comment:
this is not the function you're documenting. should be writeFile, should have data, should close parentheses.